elro/auth.py

```diff
@@ -76,6 +82,31 @@ def session(self) -> ElroConnectsCloudSessionCache | None:
         """Return the current session."""
         return self._session_cache
 
+    def _get_domain(self):
```

Comment:
We could enclose this within async_login. Store the domain when successful. When failed we keep retring each time async_login is executed.

Reply:
The problem is not really with the login, you could authenticate against every available HekrXX domain and use it accross those domains, at least a hekr.me login is usable against the hekreu.me domain. The problem relies in the device retreival, if that one is not correct you will get an empty result. So, to facilitate this the _get_domain now is called everytime and as soon as self._domain is defined, it will be returned. If not, _get_domain will try to get the correct domain from info.hekr.me